README.md

```diff
@@ -78,6 +78,9 @@ Optional high-code extensions allow almost infinite customization in a modular w
 
 (Visit the ["Why Vizro"](https://vizro.readthedocs.io/en/latest/pages/explanation/why_vizro/) section to see a more detailed explanation of Vizro use cases)
 
+Vizro also offers extensions.
+[Vizro-AI](#vizro-ai-the-innovative-extension-of-vizro) is an extension that is specifically designed to enrich existing suite of tools of Vizro,
+introducing advanced artificial intelligence features for generating, analyzing, and manipulating visuals.
```

Comment:
Maybe we just add a Packages section? Similar to this monorepo: https://github.com/microsoft/rushstack/blob/main/README.md

```suggestion
## Packages
- [vizro-core](https://vizro.readthedocs.io/en/stable/) is a high-level framework built on top of Dash, Plotly and Pydantic, which makes it easier to build advanced dashboards since it automates many of the otherwise complex and time-consuming tasks traditionally associated with designing, building and deploying front-end applications.

- [vizro-ai](https://vizro-ai.readthedocs.io/en/latest/) is an extension to [vizro-core](https://vizro.readthedocs.io/en/stable/) that is specifically designed to enrich the existing suite of tools of Vizro, introducing advanced artificial intelligence features for generating, analyzing, and manipulating visuals.
```
 And then below we also have something like this? Because currently it only shows the latest release of one of these packages here.

![Screenshot 2024-02-14 at 17 18 21](https://github.com/mckinsey/vizro/assets/90609403/b4949674-eaf5-47e0-b2f4-f19244f2a313)

Reply:
this looks like a good idea but could it probably be a table if we gets more extensions in future, do we want to put vizro core inside the table as well?

Reply:
Yes, I would put vizro-core and vizro-ai in that table